src/components/main/flight/CheckoutWithFlightScreen.js

```diff
@@ -116,15 +67,15 @@ export default class CheckoutWithFlightScreen extends React.Component {
 
   // handle remove
   handleRemove() {
-    if (this.state.treeNum != 0) {
+    if (this.state.treeNum > 1) {
       this.setState({
         treeNum: this.state.treeNum - 1,
         total_cost: this.state.total_cost - CONSTANTS.COST,
       });
     }
   }
 
-  //Calculate years to neutralize emission footprint
+  // Calculate years to neutralize emission footprint
   calcYears() {
     let tempY = this.state.footprint * 1000;
     let factor = this.state.treeNum * 18;
```

Comment:
return rounded if tempY >= 1

otherwise round it in the UI to months

Reply:
Wait, elaborate on this: the ui displays years and none are fractional years. Why are we rounding if >= 1 year to months?

Reply:
`
 // Calculate years to neutralize emission footprint
  calcYears() {	  calcYears() {
    let tempY = this.state.footprint * 1000;	  
    let factor = this.state.treeNum * 18;
    tempY /= factor;
if (years > = 1) {
    return Math.round(tempY);
}
else {
    return tempY.toFixed(2) // limits to 2 decimal places
}
`

then in the UI
`
//psuedocode
if(years > 1)
   {years} "years"
else if (years == 1)
   {years} "year"
else if (years < 0.08) // 0.08 = 1/12
   "a few days"
else  // if 0.08 <= years < 1
   {years * 12} "months"
`